src/plugins/intel_gpu/src/plugin/variable_state.cpp

```diff
@@ -70,6 +70,11 @@ void VariableState::set_state(const ov::SoPtr<ov::ITensor>& state) {
     m_layout.set_partial_shape(src_shape);
     update_device_buffer();
 
+    if (actual_size == 0) {
```

Comment:
Could you add a test for set state with empty tensor? Ideally, would be great to have LORA pattern subgraph test

Reply:
As I can see, in PR #26830 there's a LORA pattern test. I think we can reuse it (folks will make it shared)

Reply:
🫡